openid-4-verifiable-presentations-1_0.md

```diff
@@ -669,8 +669,8 @@ unknown properties.
 
 ## Credential Query {#credential_query}
 
-A Credential Query is an object representing a request for a presentation of one
-Credential.
+A Credential Query is an object representing a request for a presentation of one or more matching
+Credentials.
```

Comment:
I understand the motivation behind this change, and yet, I am not sure about the working... I think we should make it very explicit that one credential query is intended to describe exactly one credential. but, there is also a way to get copies of that one credential

Reply:
My understanding is that this is not for getting multiple copies of one credential, but instead getting multiple credentials that satisfy the credential query.

The example provided is in #298 is that the credential query asks for credentials of vct type `https://credentialrepository.com/education_degree` and the wallet can respond with multiple credentials each representing a different degree.

Reply:
ok, yes.